homeassistant/components/unifi/sensor.py

```diff
@@ -413,6 +413,20 @@ class UnifiSensorEntityDescription(
         unique_id_fn=lambda hub, obj_id: f"device_temperature-{obj_id}",
         value_fn=lambda hub, device: device.general_temperature,
     ),
+    UnifiSensorEntityDescription[Devices, Device](
```

Comment:
This should be for network clients right? Not for infrastructure

Reply:
It is infrastructure. Uplink MAC for my UDP is my modem MAC. Uplink MAC for my Switch 24 is my UDP MAC, etc.